birdhouse/components/thredds/default.env

```diff
@@ -4,6 +4,7 @@
 export THREDDS_VERSION=4.6.18-unidata-2022-01
 export THREDDS_DOCKER=pavics/thredds-docker
 export THREDDS_IMAGE="${THREDDS_DOCKER}:${THREDDS_VERSION}"
```

Comment:
single quote

Reply:
Oh wow, Misha has much better eye than me this morning :D